app/gateway/breaking-changes.md

```diff
@@ -114,14 +114,14 @@ Breaking changes in the 3.10.0.0 release.
 
 #### Node ID deprecation in `kong.conf`
 
-Manually specifying a `node_id` via Kong configuration (`kong.conf`) is deprecated. 
+Manually specifying a `node_id` via Kong configuration (`kong.conf`) is deprecated.
 The `node_id` parameter is planned to be removed in 4.x.
 
 #### AI Rate Limiting advanced plugin
 
 This release adds support for the Hugging Face provider.
 
-To import the decK configuration files that are exported from earlier versions, use the following script to transform it so that the configuration file can be compatible with the latest version:
+To import the decK configuration files that are exported from 3.9 series to earlier versions, use the following script to transform it so that the configuration file can be compatible with the latest version:
```

Comment:
```suggestion
To import the decK configuration files that are exported from the 3.9.x series to earlier versions of {{site.base_gateway}}, use the following script to transform it so that the configuration file can be compatible with the latest version:
```